src/components/BackgroundWave/BackgroundWave.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+interface Props {
+  downwards: boolean;
+}
```

Comment:
This file can be shortened by returning the SVG and replacing the d inside of path with the appropriate one depending on if you want a downwards wave or not. Same with style.

Reply:
Nice catch!